README.md

```diff
@@ -49,60 +62,69 @@ authenticator.use(microsoftStrategy);
 
 See [Microsoft docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow) for more information on `scope` and `prompt` parameters.
 
+### Organisations without multitenant
```

Comment:
I this could be more clear. Maybe something like "Applications with single-tenant authentication" but it's up to you 🙂

Reply:
You are right, change it ;)